mypyc/test-data/irbuild-lists.test

```diff
@@ -93,28 +93,28 @@ L0:
     x = r0
     return 1
 
-[case testListMultiply]
+[case testNewListTenItems]
 from typing import List
-def f(a: List[int]) -> None:
-    b = a * 2
-    b = 3 * [4]
```

Comment:
Why are you deleting this test? It looks like i served some unrelated purpose (check list multiplication)

Reply:
oops, it seems that I mistakenly overwritten it.